index.js

```diff
@@ -1,9 +1,14 @@
 'use strict';
 
+const EMBER_VERSION_WITH_JQUERY_DEPRECATION = '3.9.0-canary';
```

Comment:
I think this is subtly wrong:

```js
❯ node
> let semver = require('semver')
undefined
> semver.gte('3.9.0-beta.1', '3.9.0-canary')
false
>
```

Specifically the prerelease section is sorted _alphabetically_ such that `-beta.1` is seen as "earlier" than `-canary` 😩. This is documented in [the spec](https://semver.org/#spec-item-11) here.

```suggestion
const EMBER_VERSION_WITH_JQUERY_DEPRECATION = '3.9.0-alpha.1';
```

Reply:
Thanks! FWIW, I did play in the node console with this today, to get this prerelease handling right, but missed this edge case.

Reply:
This specific thing has trolled me so often 😩

Reply:
done